version/7_3.sh

```diff
@@ -1,31 +1,31 @@
 #!/bin/bash -e
 
 
-export PHP_VERSION="7.0.32"
+export PHP_VERSION="7.3.1"
 echo "============ Building PHP version $PHP_VERSION  =============="
-PHP_BUILD_CONFIGURE_OPTS="--with-bz2 --enable-intl --with-ldap=/usr/include" php-build -i development "$PHP_VERSION" $HOME/.phpenv/versions/7.0
+PHP_BUILD_CONFIGURE_OPTS="--with-bz2 --enable-intl --with-ldap=/usr/include" php-build -i development "$PHP_VERSION" $HOME/.phpenv/versions/7.3
 
 # Setting phpenv to PHP7.0_VERSION
-echo "============ Setting phpenv to 7.0 ============"
+echo "============ Setting phpenv to 7.3 ============"
 phpenv rehash
-phpenv global 7.0
+phpenv global 7.3
```

Comment:
I think we should introduce a `PHPENV_VERSION_ALIAS` env var and set it to `7.3`. This value is being repeated in many places and it would make sense to centralize it.

Reply:
ok